assets/app/views/settings.html

```diff
@@ -175,6 +199,6 @@ <h3 class="text-center">Memory</h3>
           </table>
         </div>
       </div>
-    </div>
+
```

Comment:
Ah, I see: line 3 is the start tag that got removed.

The `h1` above and everything after seems to have extra indentation.